components/brave_new_tab_ui/components/default/cryptoDotCom/style.ts

```diff
@@ -0,0 +1,217 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+interface StyleProps {
+  $color?: string
+  $hasSpacing?: boolean
+  as?: any
+  center?: boolean
+  clickable?: boolean
+  column?: boolean
+  disabled?: boolean
+  flex?: number
+  hasBorder?: boolean
+  hasPadding?: boolean
+  hide?: boolean
+  hideBalance?: boolean
+  hideOverflow?: boolean
+  href?: string
+  inline?: boolean
+  isActive?: boolean
+  isAsset?: boolean
+  isAuth?: boolean
+  isDeposit?: boolean
+  isDetail?: boolean
+  isFlex?: boolean
+  isLast?: boolean
+  isSmall?: boolean
+  isSummary?: boolean
+  itemsShowing?: boolean
+  light?: boolean
+  noPadding?: boolean
+  onClick?: Function
+  position?: string
+  showContent?: boolean
+  small?: boolean
+  large?: boolean
+  style?: object
+  textAlign?: string
+  textColor?: string
+  userAuthed?: boolean
+  weight?: string | number
+}
+
+const colorNameToHex = {
+  green: '#58B2A9',
+  red: '#D986A2',
+  light: 'rgba(255, 255, 255, 0.6)',
+  xlight: 'rgba(255, 255, 255, 0.3)'
+}
+
+function getColor (p: any) {
+  let colorName
+  if (!/(string|undefined)/.test(typeof p)) {
+    const keys = Object.keys(p)
+    colorName = keys.find((key) => key in colorNameToHex)
+  } else {
+    colorName = p
+  }
+  return colorNameToHex[colorName]
+}
+
+export const Text = styled<StyleProps, 'p'>('p')`
+  font-family: ${p => 'Poppins'};
```

Comment:
We shouldn't hardcode `Poppins` directly, rather use it like this: `font-family: ${p => p.theme.fontFamily.heading}`